### ISSUE TITLE
Crash while trying to sell more than one pair of binoculars

### ISSUE THREAD

**Issue by CafeineCawfee** [CleverRaven/Cataclysm-DDA#32351]
# Describe the bug

Trying to sell more than one pair of binoculars to the refugee center merchant or roaming NPCs results in a crash.

# Steps To Reproduce

1. Have more than one pair of binoculars in inventory.
2. Talk to the refugee center merchant or any NPC out in the world and initiate a trade.
3. Try selling more than one pair of binoculars.

# Expected behavior

Game shouldn't crash when trying to sell more than one pair of binoculars

# Versions and configuration

 - OS: Windows 10
 - Game Version: 0.D-5590-gabbd9f7
 - Graphics version: Tiles
 - Mods loaded: None

# Additional context

[crash.log](https://github.com/CleverRaven/Cataclysm-DDA/files/3388627/crash.log)
[debug.log](https://github.com/CleverRaven/Cataclysm-DDA/files/3388628/debug.log)
[Ruidosa.ZIP](https://github.com/CleverRaven/Cataclysm-DDA/files/3388629/Ruidosa.ZIP)

**Comment by CafeineCawfee** [comment]
Spawned in items with lenses to see if trying to pairs of said items would crash the game and I found that
-Stylish Sunglasses
-Bifocal Glasses
-Pair Of Eyeglasses
-Eclipse glasses

Also crashed the game when trying to sell them in pairs. Oddly when I tried to sell weapon scopes in pairs it didn't crash the game. Ill see if any other items with lenses crashes the game when trying to sell them in pairs.
